### PR TITLE
Bookstore Styling

### PR DESCRIPTION
**Pull request by Chimoneg27**
### For this feature I did the following 

- Created styles for the navigation, books, form and category page
- Add the progress bar circle component
- Made sure to remove linter errors
- Followed the zeppelin template design 
- Updated the user interface

### CODE REVIEW

File: src/styles/Books.css [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+  font-family: 'Roboto', sans-serif;
+  width: 90%;
+  padding: 1.5rem 1rem;
+  display: flex;
+  align-items: center;
+  background-color: #fff;
+  margin: 1.25rem auto;
+  box-shadow: 0 0 60px 0 rgba(0, 0, 0, 0.05);
+  border-radius: 4px;
+}
```
Comment (1GORDON) [inline src/styles/Books.css]: Great job @Chimoneg27 🤝

- [ ] As stated in the project requirements, kindly ensure you do not repeat the same rule. kindly create a single class for the repeating rules and re-use this class wherever needed. You may do this for the following rules. `font-family: 'Montserrat', sans-serif;`, `display: flex;`, and `font-family: 'Roboto Slab', serif;`.

File: src/styles/Form.css [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  width: 15%;
+  padding-left: 0.25rem;
+  font-size: 1rem;
+  height: 2.813rem;
+  margin: 1.188rem 2.125rem 0 0;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  font-family: 'Montserrat', sans-serif;
+  background-color: #fff;
+}
```
Comment (1GORDON) [inline src/styles/Form.css]: Great job @Chimoneg27 🤝

- [ ] As stated in the project requirements, kindly ensure you do not repeat the same rule. kindly create a single class for the repeating rules and re-use this class wherever needed. You may do this for the following rules. `font-family: 'Montserrat', sans-serif;`, and `border: solid 1px #e8e8e8;`.

File: src/styles/Books.css (hunk 1 above)
Comment (1GORDON) [inline src/styles/Books.css]: Great job @Chimoneg27 🤝

- [ ] Kindly fix this overlapping text to make your design a replica of the design provided.
overlapping-text

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @Chimoneg27 ✋

Good job so far!

## Highlights!
- Good use of classes ✔️
- Your design close to the design given ✔️
- No use of ID selectors ✔️
- No use of the important rule ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @Chimoneg27, 👋 👋  

Your project is complete! Great job for following the project requirement🥇and implementing the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




### To Highlight :1st_place_medal: 

- Descriptive Pull Request title and summary. ✔️
- Styling of the bookstore CMS project ✔️
- Zeplin Design template used for the design ✔️
- There is a good use of Git flow ✔️

## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**